workflow/rocoto/gefs_tasks.py

```diff
@@ -218,7 +218,7 @@ def _atmosoceaniceprod(self, component: str):
                      'log': f'{self.rotdir}/logs/@Y@m@d@H/{task_name}.log',
                      'maxtries': '&MAXTRIES;'}
 
-        fhrs = self._get_forecast_hours('gefs', self._configs[config])
+        fhrs = self._get_forecast_hours('gefs', self._configs[config], str(component))
```

Comment:
Component is already a str:
```suggestion
        fhrs = self._get_forecast_hours('gefs', self._configs[config], component)
```

Reply:
Done.